[2025-02-14T12:33:12.761+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T12:33:12.792+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ing_mov_20_21.run_spark_script manual__2025-02-14T12:33:09.933380+00:00 [queued]>
[2025-02-14T12:33:12.810+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ing_mov_20_21.run_spark_script manual__2025-02-14T12:33:09.933380+00:00 [queued]>
[2025-02-14T12:33:12.811+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-14T12:33:12.841+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): run_spark_script> on 2025-02-14 12:33:09.933380+00:00
[2025-02-14T12:33:12.847+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=103) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T12:33:12.849+0000] {standard_task_runner.py:72} INFO - Started process 104 to run task
[2025-02-14T12:33:12.849+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ing_mov_20_21', 'run_spark_script', 'manual__2025-02-14T12:33:09.933380+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/scriptteste.py', '--cfg-path', '/tmp/tmp1bdi8agk']
[2025-02-14T12:33:12.851+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask run_spark_script
[2025-02-14T12:33:12.946+0000] {task_command.py:467} INFO - Running <TaskInstance: ing_mov_20_21.run_spark_script manual__2025-02-14T12:33:09.933380+00:00 [running]> on host 60b47d2b4be8
[2025-02-14T12:33:13.127+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ing_mov_20_21' AIRFLOW_CTX_TASK_ID='run_spark_script' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T12:33:09.933380+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T12:33:09.933380+00:00'
[2025-02-14T12:33:13.129+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-14T12:33:13.198+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-14T12:33:13.199+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'spark-submit --packages org.apache.hadoop:hadoop-aws:3.3.6,com.amazonaws:aws-java-sdk-s3:1.12.720 /opt/bitnami/spark/scripts/ing_mov_20_21.py']
[2025-02-14T12:33:13.210+0000] {subprocess.py:99} INFO - Output:
[2025-02-14T12:33:19.370+0000] {subprocess.py:106} INFO - :: loading settings :: url = jar:file:/home/***/.local/lib/python3.12/site-packages/pyspark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-02-14T12:33:19.531+0000] {subprocess.py:106} INFO - Ivy Default Cache set to: /home/***/.ivy2/cache
[2025-02-14T12:33:19.532+0000] {subprocess.py:106} INFO - The jars for the packages stored in: /home/***/.ivy2/jars
[2025-02-14T12:33:19.538+0000] {subprocess.py:106} INFO - org.apache.hadoop#hadoop-aws added as a dependency
[2025-02-14T12:33:19.539+0000] {subprocess.py:106} INFO - com.amazonaws#aws-java-sdk-s3 added as a dependency
[2025-02-14T12:33:19.540+0000] {subprocess.py:106} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-85654ef5-cb31-487a-8861-b6f3ec5800e2;1.0
[2025-02-14T12:33:19.540+0000] {subprocess.py:106} INFO - 	confs: [default]
[2025-02-14T12:33:24.249+0000] {subprocess.py:106} INFO - 	found org.apache.hadoop#hadoop-aws;3.3.6 in central
[2025-02-14T12:33:26.194+0000] {subprocess.py:106} INFO - 	found com.amazonaws#aws-java-sdk-bundle;1.12.367 in central
[2025-02-14T12:33:28.846+0000] {subprocess.py:106} INFO - 	found org.wildfly.openssl#wildfly-openssl;1.1.3.Final in central
[2025-02-14T12:33:30.249+0000] {subprocess.py:106} INFO - 	found com.amazonaws#aws-java-sdk-s3;1.12.720 in central
[2025-02-14T12:33:30.756+0000] {subprocess.py:106} INFO - 	found com.amazonaws#aws-java-sdk-kms;1.12.720 in central
[2025-02-14T12:33:31.062+0000] {subprocess.py:106} INFO - 	found com.amazonaws#aws-java-sdk-core;1.12.720 in central
[2025-02-14T12:33:33.364+0000] {subprocess.py:106} INFO - 	found commons-logging#commons-logging;1.1.3 in central
[2025-02-14T12:33:36.427+0000] {subprocess.py:106} INFO - 	found commons-codec#commons-codec;1.15 in central
[2025-02-14T12:33:39.536+0000] {subprocess.py:106} INFO - 	found org.apache.httpcomponents#httpclient;4.5.13 in central
[2025-02-14T12:33:41.437+0000] {subprocess.py:106} INFO - 	found org.apache.httpcomponents#httpcore;4.4.13 in central
[2025-02-14T12:33:47.291+0000] {subprocess.py:106} INFO - 	found com.fasterxml.jackson.core#jackson-databind;2.12.7.2 in central
[2025-02-14T12:33:47.704+0000] {subprocess.py:106} INFO - 	found com.fasterxml.jackson.core#jackson-annotations;2.12.7 in central
[2025-02-14T12:33:48.547+0000] {subprocess.py:106} INFO - 	found com.fasterxml.jackson.core#jackson-core;2.12.7 in central
[2025-02-14T12:33:51.874+0000] {subprocess.py:106} INFO - 	found com.fasterxml.jackson.dataformat#jackson-dataformat-cbor;2.12.6 in central
[2025-02-14T12:33:52.179+0000] {subprocess.py:106} INFO - 	found joda-time#joda-time;2.12.7 in central
[2025-02-14T12:33:52.468+0000] {subprocess.py:106} INFO - 	found com.amazonaws#jmespath-java;1.12.720 in central
[2025-02-14T12:33:52.625+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/3.3.6/hadoop-aws-3.3.6.jar ...
[2025-02-14T12:33:53.324+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] org.apache.hadoop#hadoop-aws;3.3.6!hadoop-aws.jar (839ms)
[2025-02-14T12:33:53.489+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-s3/1.12.720/aws-java-sdk-s3-1.12.720.jar ...
[2025-02-14T12:33:54.067+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] com.amazonaws#aws-java-sdk-s3;1.12.720!aws-java-sdk-s3.jar (743ms)
[2025-02-14T12:33:55.963+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/1.12.367/aws-java-sdk-bundle-1.12.367.jar ...
[2025-02-14T12:36:14.013+0000] {job.py:229} INFO - Heartbeat recovered after 12.43 seconds
[2025-02-14T12:36:21.824+0000] {job.py:229} INFO - Heartbeat recovered after 11.06 seconds
[2025-02-14T12:36:30.768+0000] {job.py:229} INFO - Heartbeat recovered after 11.69 seconds
[2025-02-14T12:36:37.916+0000] {job.py:229} INFO - Heartbeat recovered after 11.03 seconds
[2025-02-14T12:37:30.634+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] com.amazonaws#aws-java-sdk-bundle;1.12.367!aws-java-sdk-bundle.jar (216565ms)
[2025-02-14T12:37:30.769+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/org/wildfly/openssl/wildfly-openssl/1.1.3.Final/wildfly-openssl-1.1.3.Final.jar ...
[2025-02-14T12:37:31.142+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] org.wildfly.openssl#wildfly-openssl;1.1.3.Final!wildfly-openssl.jar (507ms)
[2025-02-14T12:37:31.302+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-kms/1.12.720/aws-java-sdk-kms-1.12.720.jar ...
[2025-02-14T12:37:31.842+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] com.amazonaws#aws-java-sdk-kms;1.12.720!aws-java-sdk-kms.jar (699ms)
[2025-02-14T12:37:32.002+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-core/1.12.720/aws-java-sdk-core-1.12.720.jar ...
[2025-02-14T12:37:32.665+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] com.amazonaws#aws-java-sdk-core;1.12.720!aws-java-sdk-core.jar (823ms)
[2025-02-14T12:37:32.825+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/com/amazonaws/jmespath-java/1.12.720/jmespath-java-1.12.720.jar ...
[2025-02-14T12:37:32.969+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] com.amazonaws#jmespath-java;1.12.720!jmespath-java.jar (302ms)
[2025-02-14T12:37:33.104+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/commons-logging/commons-logging/1.1.3/commons-logging-1.1.3.jar ...
[2025-02-14T12:37:34.468+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] commons-logging#commons-logging;1.1.3!commons-logging.jar (1499ms)
[2025-02-14T12:37:34.602+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/commons-codec/commons-codec/1.15/commons-codec-1.15.jar ...
[2025-02-14T12:37:34.902+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] commons-codec#commons-codec;1.15!commons-codec.jar (434ms)
[2025-02-14T12:37:35.039+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar ...
[2025-02-14T12:37:35.517+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] org.apache.httpcomponents#httpclient;4.5.13!httpclient.jar (614ms)
[2025-02-14T12:37:35.680+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.12.7.2/jackson-databind-2.12.7.2.jar ...
[2025-02-14T12:37:36.443+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] com.fasterxml.jackson.core#jackson-databind;2.12.7.2!jackson-databind.jar(bundle) (924ms)
[2025-02-14T12:37:36.577+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/com/fasterxml/jackson/dataformat/jackson-dataformat-cbor/2.12.6/jackson-dataformat-cbor-2.12.6.jar ...
[2025-02-14T12:37:36.731+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] com.fasterxml.jackson.dataformat#jackson-dataformat-cbor;2.12.6!jackson-dataformat-cbor.jar(bundle) (288ms)
[2025-02-14T12:37:36.866+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/joda-time/joda-time/2.12.7/joda-time-2.12.7.jar ...
[2025-02-14T12:37:37.234+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] joda-time#joda-time;2.12.7!joda-time.jar (502ms)
[2025-02-14T12:37:37.369+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/org/apache/httpcomponents/httpcore/4.4.13/httpcore-4.4.13.jar ...
[2025-02-14T12:37:37.610+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] org.apache.httpcomponents#httpcore;4.4.13!httpcore.jar (375ms)
[2025-02-14T12:37:37.745+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.12.7/jackson-annotations-2.12.7.jar ...
[2025-02-14T12:37:37.903+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] com.fasterxml.jackson.core#jackson-annotations;2.12.7!jackson-annotations.jar(bundle) (293ms)
[2025-02-14T12:37:38.037+0000] {subprocess.py:106} INFO - downloading https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-core/2.12.7/jackson-core-2.12.7.jar ...
[2025-02-14T12:37:40.057+0000] {subprocess.py:106} INFO - 	[SUCCESSFUL ] com.fasterxml.jackson.core#jackson-core;2.12.7!jackson-core.jar(bundle) (2153ms)
[2025-02-14T12:37:40.058+0000] {subprocess.py:106} INFO - :: resolution report :: resolve 32945ms :: artifacts dl 227573ms
[2025-02-14T12:37:40.059+0000] {subprocess.py:106} INFO - 	:: modules in use:
[2025-02-14T12:37:40.060+0000] {subprocess.py:106} INFO - 	com.amazonaws#aws-java-sdk-bundle;1.12.367 from central in [default]
[2025-02-14T12:37:40.061+0000] {subprocess.py:106} INFO - 	com.amazonaws#aws-java-sdk-core;1.12.720 from central in [default]
[2025-02-14T12:37:40.062+0000] {subprocess.py:106} INFO - 	com.amazonaws#aws-java-sdk-kms;1.12.720 from central in [default]
[2025-02-14T12:37:40.063+0000] {subprocess.py:106} INFO - 	com.amazonaws#aws-java-sdk-s3;1.12.720 from central in [default]
[2025-02-14T12:37:40.065+0000] {subprocess.py:106} INFO - 	com.amazonaws#jmespath-java;1.12.720 from central in [default]
[2025-02-14T12:37:40.066+0000] {subprocess.py:106} INFO - 	com.fasterxml.jackson.core#jackson-annotations;2.12.7 from central in [default]
[2025-02-14T12:37:40.067+0000] {subprocess.py:106} INFO - 	com.fasterxml.jackson.core#jackson-core;2.12.7 from central in [default]
[2025-02-14T12:37:40.068+0000] {subprocess.py:106} INFO - 	com.fasterxml.jackson.core#jackson-databind;2.12.7.2 from central in [default]
[2025-02-14T12:37:40.073+0000] {subprocess.py:106} INFO - 	com.fasterxml.jackson.dataformat#jackson-dataformat-cbor;2.12.6 from central in [default]
[2025-02-14T12:37:40.074+0000] {subprocess.py:106} INFO - 	commons-codec#commons-codec;1.15 from central in [default]
[2025-02-14T12:37:40.119+0000] {subprocess.py:106} INFO - 	commons-logging#commons-logging;1.1.3 from central in [default]
[2025-02-14T12:37:40.119+0000] {subprocess.py:106} INFO - 	joda-time#joda-time;2.12.7 from central in [default]
[2025-02-14T12:37:40.120+0000] {subprocess.py:106} INFO - 	org.apache.hadoop#hadoop-aws;3.3.6 from central in [default]
[2025-02-14T12:37:40.121+0000] {subprocess.py:106} INFO - 	org.apache.httpcomponents#httpclient;4.5.13 from central in [default]
[2025-02-14T12:37:40.122+0000] {subprocess.py:106} INFO - 	org.apache.httpcomponents#httpcore;4.4.13 from central in [default]
[2025-02-14T12:37:40.123+0000] {subprocess.py:106} INFO - 	org.wildfly.openssl#wildfly-openssl;1.1.3.Final from central in [default]
[2025-02-14T12:37:40.124+0000] {subprocess.py:106} INFO - 	:: evicted modules:
[2025-02-14T12:37:40.124+0000] {subprocess.py:106} INFO - 	commons-logging#commons-logging;1.2 by [commons-logging#commons-logging;1.1.3] in [default]
[2025-02-14T12:37:40.125+0000] {subprocess.py:106} INFO - 	commons-codec#commons-codec;1.11 by [commons-codec#commons-codec;1.15] in [default]
[2025-02-14T12:37:40.126+0000] {subprocess.py:106} INFO - 	com.fasterxml.jackson.core#jackson-databind;2.12.6 by [com.fasterxml.jackson.core#jackson-databind;2.12.7.2] in [default]
[2025-02-14T12:37:40.126+0000] {subprocess.py:106} INFO - 	com.fasterxml.jackson.core#jackson-core;2.12.6 by [com.fasterxml.jackson.core#jackson-core;2.12.7] in [default]
[2025-02-14T12:37:40.127+0000] {subprocess.py:106} INFO - 	---------------------------------------------------------------------
[2025-02-14T12:37:40.128+0000] {subprocess.py:106} INFO - 	|                  |            modules            ||   artifacts   |
[2025-02-14T12:37:40.129+0000] {subprocess.py:106} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-02-14T12:37:40.130+0000] {subprocess.py:106} INFO - 	---------------------------------------------------------------------
[2025-02-14T12:37:40.131+0000] {subprocess.py:106} INFO - 	|      default     |   20  |   16  |   16  |   4   ||   16  |   16  |
[2025-02-14T12:37:40.132+0000] {subprocess.py:106} INFO - 	---------------------------------------------------------------------
[2025-02-14T12:37:40.133+0000] {subprocess.py:106} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-85654ef5-cb31-487a-8861-b6f3ec5800e2
[2025-02-14T12:37:40.134+0000] {subprocess.py:106} INFO - 	confs: [default]
[2025-02-14T12:37:40.673+0000] {subprocess.py:106} INFO - 	16 artifacts copied, 0 already retrieved (311600kB/602ms)
[2025-02-14T12:38:04.123+0000] {subprocess.py:106} INFO - 25/02/14 12:38:04 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-02-14T12:38:16.324+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO SparkContext: Running Spark version 3.5.4
[2025-02-14T12:38:16.325+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO SparkContext: OS info Linux, 5.15.167.4-microsoft-standard-WSL2, amd64
[2025-02-14T12:38:16.326+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO SparkContext: Java version 17.0.14
[2025-02-14T12:38:16.379+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO ResourceUtils: ==============================================================
[2025-02-14T12:38:16.380+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO ResourceUtils: No custom resources configured for spark.driver.
[2025-02-14T12:38:16.381+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO ResourceUtils: ==============================================================
[2025-02-14T12:38:16.382+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO SparkContext: Submitted application: Movies_20_21
[2025-02-14T12:38:16.431+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 2, script: , vendor: , memory -> name: memory, amount: 12288, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2025-02-14T12:38:16.453+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO ResourceProfile: Limiting resource is cpus at 2 tasks per executor
[2025-02-14T12:38:16.457+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2025-02-14T12:38:16.731+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO SecurityManager: Changing view acls to: ***
[2025-02-14T12:38:16.735+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO SecurityManager: Changing modify acls to: ***
[2025-02-14T12:38:16.737+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO SecurityManager: Changing view acls groups to:
[2025-02-14T12:38:16.740+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO SecurityManager: Changing modify acls groups to:
[2025-02-14T12:38:16.741+0000] {subprocess.py:106} INFO - 25/02/14 12:38:16 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2025-02-14T12:38:20.262+0000] {subprocess.py:106} INFO - 25/02/14 12:38:20 INFO Utils: Successfully started service 'sparkDriver' on port 32807.
[2025-02-14T12:38:20.387+0000] {subprocess.py:106} INFO - 25/02/14 12:38:20 INFO SparkEnv: Registering MapOutputTracker
[2025-02-14T12:38:20.524+0000] {subprocess.py:106} INFO - 25/02/14 12:38:20 INFO SparkEnv: Registering BlockManagerMaster
[2025-02-14T12:38:20.578+0000] {subprocess.py:106} INFO - 25/02/14 12:38:20 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2025-02-14T12:38:20.579+0000] {subprocess.py:106} INFO - 25/02/14 12:38:20 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2025-02-14T12:38:20.589+0000] {subprocess.py:106} INFO - 25/02/14 12:38:20 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2025-02-14T12:38:20.693+0000] {subprocess.py:106} INFO - 25/02/14 12:38:20 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-44992b0b-5def-40aa-bd84-4b373885bb48
[2025-02-14T12:38:20.722+0000] {subprocess.py:106} INFO - 25/02/14 12:38:20 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2025-02-14T12:38:20.747+0000] {subprocess.py:106} INFO - 25/02/14 12:38:20 INFO SparkEnv: Registering OutputCommitCoordinator
[2025-02-14T12:38:21.481+0000] {subprocess.py:106} INFO - 25/02/14 12:38:21 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2025-02-14T12:38:23.913+0000] {subprocess.py:106} INFO - 25/02/14 12:38:23 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2025-02-14T12:38:27.202+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/org.apache.hadoop_hadoop-aws-3.3.6.jar at spark://60b47d2b4be8:32807/jars/org.apache.hadoop_hadoop-aws-3.3.6.jar with timestamp 1739536696310
[2025-02-14T12:38:27.203+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-s3-1.12.720.jar at spark://60b47d2b4be8:32807/jars/com.amazonaws_aws-java-sdk-s3-1.12.720.jar with timestamp 1739536696310
[2025-02-14T12:38:27.204+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-bundle-1.12.367.jar at spark://60b47d2b4be8:32807/jars/com.amazonaws_aws-java-sdk-bundle-1.12.367.jar with timestamp 1739536696310
[2025-02-14T12:38:27.205+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/org.wildfly.openssl_wildfly-openssl-1.1.3.Final.jar at spark://60b47d2b4be8:32807/jars/org.wildfly.openssl_wildfly-openssl-1.1.3.Final.jar with timestamp 1739536696310
[2025-02-14T12:38:27.206+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-kms-1.12.720.jar at spark://60b47d2b4be8:32807/jars/com.amazonaws_aws-java-sdk-kms-1.12.720.jar with timestamp 1739536696310
[2025-02-14T12:38:27.206+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-core-1.12.720.jar at spark://60b47d2b4be8:32807/jars/com.amazonaws_aws-java-sdk-core-1.12.720.jar with timestamp 1739536696310
[2025-02-14T12:38:27.207+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/com.amazonaws_jmespath-java-1.12.720.jar at spark://60b47d2b4be8:32807/jars/com.amazonaws_jmespath-java-1.12.720.jar with timestamp 1739536696310
[2025-02-14T12:38:27.208+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/commons-logging_commons-logging-1.1.3.jar at spark://60b47d2b4be8:32807/jars/commons-logging_commons-logging-1.1.3.jar with timestamp 1739536696310
[2025-02-14T12:38:27.209+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/commons-codec_commons-codec-1.15.jar at spark://60b47d2b4be8:32807/jars/commons-codec_commons-codec-1.15.jar with timestamp 1739536696310
[2025-02-14T12:38:27.209+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/org.apache.httpcomponents_httpclient-4.5.13.jar at spark://60b47d2b4be8:32807/jars/org.apache.httpcomponents_httpclient-4.5.13.jar with timestamp 1739536696310
[2025-02-14T12:38:27.210+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/com.fasterxml.jackson.core_jackson-databind-2.12.7.2.jar at spark://60b47d2b4be8:32807/jars/com.fasterxml.jackson.core_jackson-databind-2.12.7.2.jar with timestamp 1739536696310
[2025-02-14T12:38:27.211+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/com.fasterxml.jackson.dataformat_jackson-dataformat-cbor-2.12.6.jar at spark://60b47d2b4be8:32807/jars/com.fasterxml.jackson.dataformat_jackson-dataformat-cbor-2.12.6.jar with timestamp 1739536696310
[2025-02-14T12:38:27.212+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/joda-time_joda-time-2.12.7.jar at spark://60b47d2b4be8:32807/jars/joda-time_joda-time-2.12.7.jar with timestamp 1739536696310
[2025-02-14T12:38:27.212+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/org.apache.httpcomponents_httpcore-4.4.13.jar at spark://60b47d2b4be8:32807/jars/org.apache.httpcomponents_httpcore-4.4.13.jar with timestamp 1739536696310
[2025-02-14T12:38:27.213+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/com.fasterxml.jackson.core_jackson-annotations-2.12.7.jar at spark://60b47d2b4be8:32807/jars/com.fasterxml.jackson.core_jackson-annotations-2.12.7.jar with timestamp 1739536696310
[2025-02-14T12:38:27.214+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added JAR file:///home/***/.ivy2/jars/com.fasterxml.jackson.core_jackson-core-2.12.7.jar at spark://60b47d2b4be8:32807/jars/com.fasterxml.jackson.core_jackson-core-2.12.7.jar with timestamp 1739536696310
[2025-02-14T12:38:27.214+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/org.apache.hadoop_hadoop-aws-3.3.6.jar at spark://60b47d2b4be8:32807/files/org.apache.hadoop_hadoop-aws-3.3.6.jar with timestamp 1739536696310
[2025-02-14T12:38:27.215+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/org.apache.hadoop_hadoop-aws-3.3.6.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/org.apache.hadoop_hadoop-aws-3.3.6.jar
[2025-02-14T12:38:27.524+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-s3-1.12.720.jar at spark://60b47d2b4be8:32807/files/com.amazonaws_aws-java-sdk-s3-1.12.720.jar with timestamp 1739536696310
[2025-02-14T12:38:27.525+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-s3-1.12.720.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/com.amazonaws_aws-java-sdk-s3-1.12.720.jar
[2025-02-14T12:38:27.530+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-bundle-1.12.367.jar at spark://60b47d2b4be8:32807/files/com.amazonaws_aws-java-sdk-bundle-1.12.367.jar with timestamp 1739536696310
[2025-02-14T12:38:27.531+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-bundle-1.12.367.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/com.amazonaws_aws-java-sdk-bundle-1.12.367.jar
[2025-02-14T12:38:27.771+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/org.wildfly.openssl_wildfly-openssl-1.1.3.Final.jar at spark://60b47d2b4be8:32807/files/org.wildfly.openssl_wildfly-openssl-1.1.3.Final.jar with timestamp 1739536696310
[2025-02-14T12:38:27.772+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/org.wildfly.openssl_wildfly-openssl-1.1.3.Final.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/org.wildfly.openssl_wildfly-openssl-1.1.3.Final.jar
[2025-02-14T12:38:27.777+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-kms-1.12.720.jar at spark://60b47d2b4be8:32807/files/com.amazonaws_aws-java-sdk-kms-1.12.720.jar with timestamp 1739536696310
[2025-02-14T12:38:27.778+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-kms-1.12.720.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/com.amazonaws_aws-java-sdk-kms-1.12.720.jar
[2025-02-14T12:38:27.785+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-core-1.12.720.jar at spark://60b47d2b4be8:32807/files/com.amazonaws_aws-java-sdk-core-1.12.720.jar with timestamp 1739536696310
[2025-02-14T12:38:27.786+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/com.amazonaws_aws-java-sdk-core-1.12.720.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/com.amazonaws_aws-java-sdk-core-1.12.720.jar
[2025-02-14T12:38:27.793+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/com.amazonaws_jmespath-java-1.12.720.jar at spark://60b47d2b4be8:32807/files/com.amazonaws_jmespath-java-1.12.720.jar with timestamp 1739536696310
[2025-02-14T12:38:27.796+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/com.amazonaws_jmespath-java-1.12.720.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/com.amazonaws_jmespath-java-1.12.720.jar
[2025-02-14T12:38:27.799+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/commons-logging_commons-logging-1.1.3.jar at spark://60b47d2b4be8:32807/files/commons-logging_commons-logging-1.1.3.jar with timestamp 1739536696310
[2025-02-14T12:38:27.800+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/commons-logging_commons-logging-1.1.3.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/commons-logging_commons-logging-1.1.3.jar
[2025-02-14T12:38:27.805+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/commons-codec_commons-codec-1.15.jar at spark://60b47d2b4be8:32807/files/commons-codec_commons-codec-1.15.jar with timestamp 1739536696310
[2025-02-14T12:38:27.806+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/commons-codec_commons-codec-1.15.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/commons-codec_commons-codec-1.15.jar
[2025-02-14T12:38:27.812+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/org.apache.httpcomponents_httpclient-4.5.13.jar at spark://60b47d2b4be8:32807/files/org.apache.httpcomponents_httpclient-4.5.13.jar with timestamp 1739536696310
[2025-02-14T12:38:27.813+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/org.apache.httpcomponents_httpclient-4.5.13.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/org.apache.httpcomponents_httpclient-4.5.13.jar
[2025-02-14T12:38:27.819+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/com.fasterxml.jackson.core_jackson-databind-2.12.7.2.jar at spark://60b47d2b4be8:32807/files/com.fasterxml.jackson.core_jackson-databind-2.12.7.2.jar with timestamp 1739536696310
[2025-02-14T12:38:27.820+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/com.fasterxml.jackson.core_jackson-databind-2.12.7.2.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/com.fasterxml.jackson.core_jackson-databind-2.12.7.2.jar
[2025-02-14T12:38:27.827+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/com.fasterxml.jackson.dataformat_jackson-dataformat-cbor-2.12.6.jar at spark://60b47d2b4be8:32807/files/com.fasterxml.jackson.dataformat_jackson-dataformat-cbor-2.12.6.jar with timestamp 1739536696310
[2025-02-14T12:38:27.828+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/com.fasterxml.jackson.dataformat_jackson-dataformat-cbor-2.12.6.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/com.fasterxml.jackson.dataformat_jackson-dataformat-cbor-2.12.6.jar
[2025-02-14T12:38:27.832+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/joda-time_joda-time-2.12.7.jar at spark://60b47d2b4be8:32807/files/joda-time_joda-time-2.12.7.jar with timestamp 1739536696310
[2025-02-14T12:38:27.833+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/joda-time_joda-time-2.12.7.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/joda-time_joda-time-2.12.7.jar
[2025-02-14T12:38:27.987+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/org.apache.httpcomponents_httpcore-4.4.13.jar at spark://60b47d2b4be8:32807/files/org.apache.httpcomponents_httpcore-4.4.13.jar with timestamp 1739536696310
[2025-02-14T12:38:27.988+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/org.apache.httpcomponents_httpcore-4.4.13.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/org.apache.httpcomponents_httpcore-4.4.13.jar
[2025-02-14T12:38:27.992+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/com.fasterxml.jackson.core_jackson-annotations-2.12.7.jar at spark://60b47d2b4be8:32807/files/com.fasterxml.jackson.core_jackson-annotations-2.12.7.jar with timestamp 1739536696310
[2025-02-14T12:38:27.992+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/com.fasterxml.jackson.core_jackson-annotations-2.12.7.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/com.fasterxml.jackson.core_jackson-annotations-2.12.7.jar
[2025-02-14T12:38:27.996+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO SparkContext: Added file file:///home/***/.ivy2/jars/com.fasterxml.jackson.core_jackson-core-2.12.7.jar at spark://60b47d2b4be8:32807/files/com.fasterxml.jackson.core_jackson-core-2.12.7.jar with timestamp 1739536696310
[2025-02-14T12:38:27.997+0000] {subprocess.py:106} INFO - 25/02/14 12:38:27 INFO Utils: Copying /home/***/.ivy2/jars/com.fasterxml.jackson.core_jackson-core-2.12.7.jar to /tmp/spark-9dda0699-fc52-4b91-94ba-58c69e41eae2/userFiles-8b28bcce-379a-4ae1-8afb-2b4b4e474836/com.fasterxml.jackson.core_jackson-core-2.12.7.jar
[2025-02-14T12:38:34.134+0000] {subprocess.py:106} INFO - 25/02/14 12:38:34 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2025-02-14T12:38:35.753+0000] {subprocess.py:106} INFO - 25/02/14 12:38:35 INFO TransportClientFactory: Successfully created connection to spark/172.18.0.5:7077 after 1148 ms (0 ms spent in bootstraps)
[2025-02-14T12:38:43.455+0000] {subprocess.py:106} INFO - 25/02/14 12:38:43 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20250214123843-0000
[2025-02-14T12:38:43.944+0000] {subprocess.py:106} INFO - 25/02/14 12:38:43 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 37035.
[2025-02-14T12:38:43.945+0000] {subprocess.py:106} INFO - 25/02/14 12:38:43 INFO NettyBlockTransferService: Server created on 60b47d2b4be8:37035
[2025-02-14T12:38:44.136+0000] {subprocess.py:106} INFO - 25/02/14 12:38:44 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2025-02-14T12:38:44.491+0000] {subprocess.py:106} INFO - 25/02/14 12:38:44 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20250214123843-0000/0 on worker-20250214122831-172.18.0.2-40721 (172.18.0.2:40721) with 2 core(s)
[2025-02-14T12:38:44.494+0000] {subprocess.py:106} INFO - 25/02/14 12:38:44 INFO StandaloneSchedulerBackend: Granted executor ID app-20250214123843-0000/0 on hostPort 172.18.0.2:40721 with 2 core(s), 12.0 GiB RAM
[2025-02-14T12:38:44.495+0000] {subprocess.py:106} INFO - 25/02/14 12:38:44 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20250214123843-0000/1 on worker-20250214122831-172.18.0.6-41581 (172.18.0.6:41581) with 2 core(s)
[2025-02-14T12:38:44.496+0000] {subprocess.py:106} INFO - 25/02/14 12:38:44 INFO StandaloneSchedulerBackend: Granted executor ID app-20250214123843-0000/1 on hostPort 172.18.0.6:41581 with 2 core(s), 12.0 GiB RAM
[2025-02-14T12:38:44.807+0000] {subprocess.py:106} INFO - 25/02/14 12:38:44 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 60b47d2b4be8, 37035, None)
[2025-02-14T12:38:44.861+0000] {subprocess.py:106} INFO - 25/02/14 12:38:44 INFO BlockManagerMasterEndpoint: Registering block manager 60b47d2b4be8:37035 with 434.4 MiB RAM, BlockManagerId(driver, 60b47d2b4be8, 37035, None)
[2025-02-14T12:38:44.910+0000] {subprocess.py:106} INFO - 25/02/14 12:38:44 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 60b47d2b4be8, 37035, None)
[2025-02-14T12:38:44.913+0000] {subprocess.py:106} INFO - 25/02/14 12:38:44 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 60b47d2b4be8, 37035, None)
[2025-02-14T12:38:51.830+0000] {job.py:229} INFO - Heartbeat recovered after 13.87 seconds
[2025-02-14T12:38:54.833+0000] {subprocess.py:106} INFO - 25/02/14 12:38:54 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250214123843-0000/0 is now RUNNING
[2025-02-14T12:38:54.835+0000] {subprocess.py:106} INFO - 25/02/14 12:38:54 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250214123843-0000/1 is now RUNNING
[2025-02-14T12:38:57.062+0000] {job.py:229} INFO - Heartbeat recovered after 11.17 seconds
[2025-02-14T12:39:05.157+0000] {subprocess.py:106} INFO - 25/02/14 12:39:05 INFO AsyncEventQueue: Process of event SparkListenerResourceProfileAdded(Profile: id = 0, executor resources: cores -> name: cores, amount: 2, script: , vendor: ,memory -> name: memory, amount: 12288, script: , vendor: ,offHeap -> name: offHeap, amount: 0, script: , vendor: , task resources: cpus -> name: cpus, amount: 1.0) by listener AppStatusListener took 2.08814175s.
[2025-02-14T12:39:07.171+0000] {subprocess.py:106} INFO - 25/02/14 12:39:07 INFO AsyncEventQueue: Process of event SparkListenerBlockManagerAdded(1739536724859,BlockManagerId(driver, 60b47d2b4be8, 37035, None),455501414,Some(455501414),Some(0)) by listener AppStatusListener took 2.012850879s.
[2025-02-14T12:39:07.174+0000] {subprocess.py:106} INFO - 25/02/14 12:39:07 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2025-02-14T12:39:26.017+0000] {subprocess.py:106} INFO - 25/02/14 12:39:26 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.18.0.6:43742) with ID 1,  ResourceProfileId 0
[2025-02-14T12:39:26.736+0000] {subprocess.py:106} INFO - 25/02/14 12:39:26 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.18.0.2:46492) with ID 0,  ResourceProfileId 0
[2025-02-14T12:39:27.707+0000] {subprocess.py:106} INFO - 25/02/14 12:39:27 INFO BlockManagerMasterEndpoint: Registering block manager 172.18.0.2:35339 with 7.0 GiB RAM, BlockManagerId(0, 172.18.0.2, 35339, None)
[2025-02-14T12:39:27.709+0000] {subprocess.py:106} INFO - 25/02/14 12:39:27 INFO BlockManagerMasterEndpoint: Registering block manager 172.18.0.6:37611 with 7.0 GiB RAM, BlockManagerId(1, 172.18.0.6, 37611, None)
[2025-02-14T12:40:48.367+0000] {subprocess.py:106} INFO - Traceback (most recent call last):
[2025-02-14T12:40:48.369+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/urllib3/connection.py", line 198, in _new_conn
[2025-02-14T12:40:48.370+0000] {subprocess.py:106} INFO -     sock = connection.create_connection(
[2025-02-14T12:40:48.371+0000] {subprocess.py:106} INFO -            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.371+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/urllib3/util/connection.py", line 60, in create_connection
[2025-02-14T12:40:48.372+0000] {subprocess.py:106} INFO -     for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
[2025-02-14T12:40:48.372+0000] {subprocess.py:106} INFO -                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.373+0000] {subprocess.py:106} INFO -   File "/usr/local/lib/python3.12/socket.py", line 978, in getaddrinfo
[2025-02-14T12:40:48.374+0000] {subprocess.py:106} INFO -     for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
[2025-02-14T12:40:48.374+0000] {subprocess.py:106} INFO -                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.375+0000] {subprocess.py:106} INFO - socket.gaierror: [Errno -3] Temporary failure in name resolution
[2025-02-14T12:40:48.375+0000] {subprocess.py:106} INFO - 
[2025-02-14T12:40:48.376+0000] {subprocess.py:106} INFO - The above exception was the direct cause of the following exception:
[2025-02-14T12:40:48.377+0000] {subprocess.py:106} INFO - 
[2025-02-14T12:40:48.378+0000] {subprocess.py:106} INFO - Traceback (most recent call last):
[2025-02-14T12:40:48.378+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 787, in urlopen
[2025-02-14T12:40:48.379+0000] {subprocess.py:106} INFO -     response = self._make_request(
[2025-02-14T12:40:48.380+0000] {subprocess.py:106} INFO -                ^^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.380+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 488, in _make_request
[2025-02-14T12:40:48.381+0000] {subprocess.py:106} INFO -     raise new_e
[2025-02-14T12:40:48.382+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 464, in _make_request
[2025-02-14T12:40:48.383+0000] {subprocess.py:106} INFO -     self._validate_conn(conn)
[2025-02-14T12:40:48.383+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 1093, in _validate_conn
[2025-02-14T12:40:48.384+0000] {subprocess.py:106} INFO -     conn.connect()
[2025-02-14T12:40:48.385+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/urllib3/connection.py", line 704, in connect
[2025-02-14T12:40:48.386+0000] {subprocess.py:106} INFO -     self.sock = sock = self._new_conn()
[2025-02-14T12:40:48.386+0000] {subprocess.py:106} INFO -                        ^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.387+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/urllib3/connection.py", line 205, in _new_conn
[2025-02-14T12:40:48.388+0000] {subprocess.py:106} INFO -     raise NameResolutionError(self.host, self, e) from e
[2025-02-14T12:40:48.389+0000] {subprocess.py:106} INFO - urllib3.exceptions.NameResolutionError: <urllib3.connection.HTTPSConnection object at 0x7fa68b056930>: Failed to resolve 'api.themoviedb.org' ([Errno -3] Temporary failure in name resolution)
[2025-02-14T12:40:48.390+0000] {subprocess.py:106} INFO - 
[2025-02-14T12:40:48.390+0000] {subprocess.py:106} INFO - The above exception was the direct cause of the following exception:
[2025-02-14T12:40:48.391+0000] {subprocess.py:106} INFO - 
[2025-02-14T12:40:48.392+0000] {subprocess.py:106} INFO - Traceback (most recent call last):
[2025-02-14T12:40:48.392+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/requests/adapters.py", line 667, in send
[2025-02-14T12:40:48.393+0000] {subprocess.py:106} INFO -     resp = conn.urlopen(
[2025-02-14T12:40:48.394+0000] {subprocess.py:106} INFO -            ^^^^^^^^^^^^^
[2025-02-14T12:40:48.395+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 841, in urlopen
[2025-02-14T12:40:48.396+0000] {subprocess.py:106} INFO -     retries = retries.increment(
[2025-02-14T12:40:48.397+0000] {subprocess.py:106} INFO -               ^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.398+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/urllib3/util/retry.py", line 519, in increment
[2025-02-14T12:40:48.399+0000] {subprocess.py:106} INFO -     raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
[2025-02-14T12:40:48.399+0000] {subprocess.py:106} INFO -     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.400+0000] {subprocess.py:106} INFO - urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.themoviedb.org', port=443): Max retries exceeded with url: /3/discover/movie?include_adult=false&include_video=false&language=en-US&page=78&sort_by=popularity.desc&primary_release_date.gte=2020-01-01&primary_release_date.lte=2020-01-01 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fa68b056930>: Failed to resolve 'api.themoviedb.org' ([Errno -3] Temporary failure in name resolution)"))
[2025-02-14T12:40:48.401+0000] {subprocess.py:106} INFO - 
[2025-02-14T12:40:48.401+0000] {subprocess.py:106} INFO - During handling of the above exception, another exception occurred:
[2025-02-14T12:40:48.402+0000] {subprocess.py:106} INFO - 
[2025-02-14T12:40:48.403+0000] {subprocess.py:106} INFO - Traceback (most recent call last):
[2025-02-14T12:40:48.403+0000] {subprocess.py:106} INFO -   File "/opt/bitnami/spark/scripts/ing_mov_20_21.py", line 88, in <module>
[2025-02-14T12:40:48.404+0000] {subprocess.py:106} INFO -     response = requests.get(url, headers=headers, params=params)
[2025-02-14T12:40:48.404+0000] {subprocess.py:106} INFO -                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.405+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/requests/api.py", line 73, in get
[2025-02-14T12:40:48.405+0000] {subprocess.py:106} INFO -     return request("get", url, params=params, **kwargs)
[2025-02-14T12:40:48.406+0000] {subprocess.py:106} INFO -            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.407+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/requests/api.py", line 59, in request
[2025-02-14T12:40:48.408+0000] {subprocess.py:106} INFO -     return session.request(method=method, url=url, **kwargs)
[2025-02-14T12:40:48.409+0000] {subprocess.py:106} INFO -            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.410+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/requests/sessions.py", line 589, in request
[2025-02-14T12:40:48.411+0000] {subprocess.py:106} INFO -     resp = self.send(prep, **send_kwargs)
[2025-02-14T12:40:48.412+0000] {subprocess.py:106} INFO -            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.413+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/requests/sessions.py", line 703, in send
[2025-02-14T12:40:48.414+0000] {subprocess.py:106} INFO -     r = adapter.send(request, **kwargs)
[2025-02-14T12:40:48.414+0000] {subprocess.py:106} INFO -         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-02-14T12:40:48.415+0000] {subprocess.py:106} INFO -   File "/home/***/.local/lib/python3.12/site-packages/requests/adapters.py", line 700, in send
[2025-02-14T12:40:48.416+0000] {subprocess.py:106} INFO -     raise ConnectionError(e, request=request)
[2025-02-14T12:40:48.416+0000] {subprocess.py:106} INFO - requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.themoviedb.org', port=443): Max retries exceeded with url: /3/discover/movie?include_adult=false&include_video=false&language=en-US&page=78&sort_by=popularity.desc&primary_release_date.gte=2020-01-01&primary_release_date.lte=2020-01-01 (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fa68b056930>: Failed to resolve 'api.themoviedb.org' ([Errno -3] Temporary failure in name resolution)"))
[2025-02-14T12:40:48.501+0000] {subprocess.py:106} INFO - 25/02/14 12:40:48 INFO SparkContext: Invoking stop() from shutdown hook
[2025-02-14T12:40:48.502+0000] {subprocess.py:106} INFO - 25/02/14 12:40:48 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2025-02-14T12:41:16.195+0000] {job.py:229} INFO - Heartbeat recovered after 103.92 seconds
[2025-02-14T12:41:18.500+0000] {subprocess.py:106} INFO - 25/02/14 12:41:18 WARN ShutdownHookManager: ShutdownHook '' timeout, java.util.concurrent.TimeoutException
[2025-02-14T12:41:18.501+0000] {subprocess.py:106} INFO - java.util.concurrent.TimeoutException
[2025-02-14T12:41:18.501+0000] {subprocess.py:106} INFO - 	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
[2025-02-14T12:41:18.502+0000] {subprocess.py:106} INFO - 	at org.apache.hadoop.util.ShutdownHookManager.executeShutdown(ShutdownHookManager.java:124)
[2025-02-14T12:41:18.503+0000] {subprocess.py:106} INFO - 	at org.apache.hadoop.util.ShutdownHookManager$1.run(ShutdownHookManager.java:95)
[2025-02-14T12:41:26.207+0000] {job.py:239} ERROR - Job heartbeat failed with error
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
           ^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
          ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 690, in __connect
    with util.safe_reraise():
         ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.OperationalError: could not translate host name "postgres" to address: Temporary failure in name resolution


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/jobs/job.py", line 233, in heartbeat
    heartbeat_callback(session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/jobs/local_task_job_runner.py", line 284, in heartbeat_callback
    self.task_instance.refresh_from_db()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 2360, in refresh_from_db
    _refresh_from_db(task_instance=self, session=session, lock_for_update=lock_for_update)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 873, in _refresh_from_db
    ti = TaskInstance.get_task_instance(
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 166, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 2346, in get_task_instance
    return query.one_or_none()
           ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/orm/query.py", line 2850, in one_or_none
    return self._iter().one_or_none()
           ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/orm/query.py", line 2916, in _iter
    result = self.session.execute(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 1716, in execute
    conn = self._connection_for_bind(bind)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 1555, in _connection_for_bind
    return self._transaction._connection_for_bind(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 750, in _connection_for_bind
    conn = bind.connect()
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/future/engine.py", line 412, in connect
    return super(Engine, self).connect()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 3325, in connect
    return self._connection_cls(self, close_with_result=close_with_result)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
    else engine.raw_connection()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 3404, in raw_connection
    return self._wrap_pool_connect(self.pool.connect, _connection)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 3374, in _wrap_pool_connect
    Connection._handle_dbapi_exception_noconnection(
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 2208, in _handle_dbapi_exception_noconnection
    util.raise_(
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
           ^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
          ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 690, in __connect
    with util.safe_reraise():
         ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2025-02-14T12:41:26.213+0000] {job.py:254} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
[2025-02-14T12:41:48.499+0000] {subprocess.py:106} INFO - 25/02/14 12:41:48 WARN ShutdownHookManager: ShutdownHook 'ClientFinalizer' timeout, java.util.concurrent.TimeoutException
[2025-02-14T12:41:48.501+0000] {subprocess.py:106} INFO - java.util.concurrent.TimeoutException
[2025-02-14T12:41:48.503+0000] {subprocess.py:106} INFO - 	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
[2025-02-14T12:41:48.504+0000] {subprocess.py:106} INFO - 	at org.apache.hadoop.util.ShutdownHookManager.executeShutdown(ShutdownHookManager.java:124)
[2025-02-14T12:41:48.505+0000] {subprocess.py:106} INFO - 	at org.apache.hadoop.util.ShutdownHookManager$1.run(ShutdownHookManager.java:95)
[2025-02-14T12:41:58.796+0000] {job.py:229} INFO - Heartbeat recovered after 141.24 seconds
[2025-02-14T12:46:39.208+0000] {subprocess.py:106} INFO - 25/02/14 12:46:39 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250214123843-0000/1 is now LOST (worker lost: Not receiving heartbeat for 60 seconds)
[2025-02-14T12:46:39.210+0000] {subprocess.py:106} INFO - 25/02/14 12:46:39 INFO StandaloneSchedulerBackend: Executor app-20250214123843-0000/1 removed: worker lost: Not receiving heartbeat for 60 seconds
[2025-02-14T12:46:55.335+0000] {job.py:229} INFO - Heartbeat recovered after 324.07 seconds
[2025-02-14T12:47:21.886+0000] {job.py:229} INFO - Heartbeat recovered after 318.04 seconds
[2025-02-14T12:47:59.940+0000] {subprocess.py:106} INFO - 25/02/14 12:47:59 ERROR ShutdownHookManager: ShutdownHookManager shutdown forcefully after 30 seconds.
[2025-02-14T12:48:02.616+0000] {job.py:229} INFO - Heartbeat recovered after 62.23 seconds
[2025-02-14T12:48:03.135+0000] {subprocess.py:110} INFO - Command exited with return code 1
[2025-02-14T12:48:03.147+0000] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-02-14T12:48:03.155+0000] {taskinstance.py:1226} INFO - Marking task as UP_FOR_RETRY. dag_id=ing_mov_20_21, task_id=run_spark_script, run_id=manual__2025-02-14T12:33:09.933380+00:00, execution_date=20250214T123309, start_date=20250214T123312, end_date=20250214T124803
[2025-02-14T12:48:46.919+0000] {job.py:229} INFO - Heartbeat recovered after 79.98 seconds
[2025-02-14T12:48:50.147+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-14T12:48:50.148+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 3 for task run_spark_script (Bash command failed. The command returned a non-zero exit code 1.; 104)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-02-14T12:48:50.194+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-02-14T12:48:50.216+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-14T12:48:51.770+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-14T12:48:52.899+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
